xbmc/utils/FileUtils.cpp

```diff
@@ -7,24 +7,26 @@
  */
 
 #include "FileUtils.h"
-#include "ServiceBroker.h"
-#include "guilib/GUIKeyboardFactory.h"
-#include "utils/log.h"
-#include "guilib/LocalizeStrings.h"
-#include "JobManager.h"
```

Comment:
All this churn is because @jenkins4kodi told me to change it: https://jenkins.kodi.tv/job/BuildMulti-PR/22262/artifact/PR21113.diff